settings.yaml.template

```diff
@@ -73,7 +73,8 @@ AWS:
         ACCESS_KEY:
         SECRET_KEY:
         # Multiple regions can be added like ["ap-south-1", "us-west-2", "us-west-1"] or ["all"] for all regions
-        REGIONS: []
+        REGIONS: [] # Cleanup regions
+        CLIENT_REGION: "us-east-1"
```

Comment:
For retrieving OCPs resources using the `Resource Explorer` tool, we need to define a client region, because `Rsource Explorer` allows querying resources only against an authorized client, and the client must relate to a certain region (the default is `us-east-1`).

**In summary:** 
The `REGIONS` is a list of "cleanup regions" as currently implemented, and the `CLIENT_REGION` is only for setting the client for the OCP leftovers.
**See example:**
https://github.com/oharan2/wrapanapi/blob/list_ocp_resources/wrapanapi/systems/ec2.py#L1891-L1893